pkg/daemon/daemon.go

```diff
@@ -689,6 +708,61 @@ func (dn *Daemon) applySSHAccessedAnnotation() error {
 	return nil
 }
 
+func (dn *Daemon) runConfigDriftMonitor(stopCh <-chan struct{}, exitCh chan<- error) {
+	// Set this to 10 seconds for testing purposes
+	interval := 10 * time.Second
+
+	glog.Infof("Starting Config Drift Watcher, will validate every %s", interval)
+
+	for {
+		select {
+		case <-stopCh:
+			return
+		case <-time.After(interval):
+			if err := dn.validateCurrentOnDiskState(); err != nil {
+				// TODO: Use better error handling here
+				if strings.Contains(err.Error(), "on-disk state mismatch") {
+					dn.updateErrorState(err)
+				} else {
+					exitCh <- err
+				}
+			}
+		}
+	}
+}
+
+// Not sure if this is actually needed or if I can use getCurrentConfigOnDisk instead?
```

Comment:
Technically possible to have difference. My interpretation is that the on-disk version should always be correct, although in practice it doesn't seem to be that rigorous.